netdeploy/network.go

```diff
@@ -81,6 +81,13 @@ func CreateNetworkFromTemplate(name, rootDir, templateFile, binDir string, impor
 		return n, err
 	}
 
+	if n.cfg.Name == "" {
+		n.cfg.Name = template.Genesis.NetworkName
```

Comment:
just below here on line 97 `generateGenesisAndWallets` takes the `name` var, so either the template value needs to move back into that var as well, or below should use n.cfg.Name